object/cert.go

```diff
@@ -37,6 +87,8 @@ type Cert struct {
 	PrivateKey             string `xorm:"mediumtext" json:"privateKey"`
 	AuthorityPublicKey     string `xorm:"mediumtext" json:"authorityPublicKey"`
 	AuthorityRootPublicKey string `xorm:"mediumtext" json:"authorityRootPublicKey"`
+
+	CACertificate string `xorm:"mediumtext" json:"caCertificate"`
```

Comment:
Why not use `Certificate`?

Reply:
You can switch the "scope" to JWT and then the certificate and key will be incompatible.
This can lead to errors that are difficult to detect.

Reply:
That's the wrong scope's problem, not a reason to add a duplicated field

Reply:
@hsluoyz but the ca certificate is used to verify ssl connection, not to sign data. If you are using  certificate scope to set up ssl ldap fill unused fields. It will not be obvious to the user